### PR TITLE
Miscellaneous minor changes

### PR DESCRIPTION
**Pull request by ddorwin**
Includes addressing issues from previous PRs.

### CODE REVIEW

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -2499,11 +2498,10 @@
         <section id="monitor-cdm">
           <h4>Monitor for CDM State Changes</h4>
           <p>
-            The Monitor for CDM State Changes algorithm executes steps required when various aspects of CDM state change. It is run for each
-            <a>MediaKeySession</a> object in parallel to the main event loop but not in parallel to other procedures defined in this
-            specification that are run in parallel to the main event loop.
+            The Monitor for CDM State Changes algorithm executes steps required when various aspects of CDM state change.
           </p>
           <p class="note">This algorithm only applies to CDM state changes that are not covered by other algorithms.  For example, <a def-id="update"></a> may result in messages, key status changes and/or expiration changes, but those are all handled within that algorithm.</p>
+          <p class="note">The algorithm is always run in parallel to the main event loop.</p>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: This is consistent with algorithms and removes duplication of line 1606.

File: encrypted-media-respec.html [hunk 2]
```diff
@@ -3942,7 +3941,7 @@
               <li><p>MUST be sent to a URL selected independently of any origin and application.</p></li>
             </ul>
             <p>
-              Implementations MUST NOT expose, even in encrypted form, <a def-id="origin">origin(s)</a>-, origin- or application-specific information, or values that are <a def-id="associable"></a> with origin(s) to centralized servers since this could create a central record of all origins visited by a user or device.
+              Implementations MUST NOT expose, even in encrypted form, <a def-id="origin">origin(s)</a>, origin- or application-specific information, or values that are <a def-id="associable"></a> with origin(s) to centralized servers since this could create a central record of all origins visited by a user or device.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: Fixes issue identified in https://github.com/w3c/encrypted-media/issues/272#issuecomment-234428862.

File: encrypted-media-respec.html [hunk 3]
```diff
@@ -3908,16 +3906,17 @@
...
             The resulting identifier(s) MUST be <a def-id="non-associable-by-application"></a> and <a href="#uses-distinctive-identifiers">use of them</a> MUST only be exposed to a <a href="#per-origin-per-profile-identifiers">single origin from a single profile</a>.
             This process MAY be performed multiple times, such as after identifier(s) are <a href="#allow-identifiers-cleared">cleared</a>.
           </p>
-          <p>This process MUST be performed either <a href="#direct-individualization">directly by the user agent</a> or <a href="#app-assisted-individualization">through the application</a> depending on the CDM implementation.
+          <p>This process MUST be performed either <a href="#direct-individualization">directly by the user agent</a> or <a href="#app-assisted-individualization">through the application</a>.
             The mechanisms, flow, and restrictions for the two types of individualization are different, as described in the following sections.
+            Which method is used depends on the CDM implementation and application of the requirements of this specification, especially those below.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: Addresses https://github.com/w3c/encrypted-media/pull/281#discussion_r71814134.

**Comment by mwatson2** [comment]
LGTM

**Pull request merged**